units/user/systemd-exit.service

```diff
@@ -14,3 +14,6 @@ DefaultDependencies=no
 Requires=shutdown.target
 After=shutdown.target
 SuccessAction=exit-force
+
+[Service]
+Slice=-.slice
```

Comment:
I think this fix is actually right, but could you add a comment why you added this to the unit file